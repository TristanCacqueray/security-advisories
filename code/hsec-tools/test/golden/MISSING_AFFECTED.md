```toml
[advisory]
id = "HSEC-0000-0000"
cwe = []
```

## Title
