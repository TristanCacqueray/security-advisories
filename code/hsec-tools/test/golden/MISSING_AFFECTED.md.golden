Left
    ( AdvisoryError
        ( MissingKey "affected" ) "MissingKey "affected""
    )
